src/AWS.Deploy.Orchestration/DeploymentCommands/BeanstalkEnvironmentDeploymentCommand.cs

```diff
@@ -37,19 +37,21 @@ public async Task ExecuteAsync(Orchestrator orchestrator, CloudApplication cloud
                 throw new AWSResourceNotFoundException(DeployToolErrorCode.FailedToFindElasticBeanstalkApplication, message);
             }
 
-            orchestrator._interactiveService.LogMessageLine($"Initiating deployment to {cloudApplication.DisplayName}...");
+            orchestrator._interactiveService.LogSectionStart($"Create application version", "Uploading deployment bundle to S3 and create an Elastic Beanstalk application version");
```

Comment:
nit: Most other `LogSectionStart` calls are using `-ing` on the verb, but here it's just "Create" and "Deploy".

I think it'd be nice polish if we were consistent throughout.

Reply:
> Current behavior will stream the headers as they come in so in terms of UI, the toolkit will add new sections adhoc which wouldn't give the user a sense of what the overall process is going to be like. This doesn't have to be part of this PR, but it's worth thinking of a way to outline major sections like Configuration, Deployment Bundle, Deployment (things that we know will happen for sure regardless of recipe) and the rest of the recipe dependent headers will be streamed in.

Yes, Chris and I have discussed this. But since our system is so dynamic we don't currently have a way in our system to precompute all of the stages. That can be something we possibly do in the future.